cluster/cluster.go

```diff
@@ -246,10 +250,13 @@ func getLocalAdminConfigWithNewAddress(localConfigPath, cpAddress string) string
 		string(config.KeyData))
 }
 
-func (c *Cluster) ApplyRBACResources() error {
+func (c *Cluster) ApplyAuthzResources() error {
 	if err := authz.ApplyJobDeployerServiceAccount(c.LocalKubeConfigPath); err != nil {
 		return fmt.Errorf("Failed to apply the ServiceAccount needed for job execution: %v", err)
 	}
+	if c.Authorization.Mode == NoneAuthorizationMode {
```

Comment:
shouldn't we check that before sending ApplyJobDeployerServiceAccount?

Reply:
No we need that anyway because it's added to the job manifest. If it's removed, the job will break.